packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -491,6 +500,14 @@ class AndroidCameraCameraX extends CameraPlatform {
   /// Sets the flash mode for the selected camera.
```

Comment:
nit: Might be worth adding to the documentation here that this method combines the notion of flash and torch from

https://developer.android.com/reference/androidx/camera/core/ImageCapture and
https://developer.android.com/reference/androidx/camera/core/CameraControl#enableTorch(boolean).